cmd/launcher/internal/updater/updater.go

```diff
@@ -79,17 +97,22 @@ func NewUpdater(
 // updater allows us to mock *autoupdate.Updater during testing
 type updater interface {
 	Run(opts ...tuf.Option) (stop func(), err error)
+	RollingErrorCount() int
 }
 
 type updaterCmd struct {
```

Comment:
For what it's worth, I don't think I have an opinion about whether this model is at all correct. It think it carries a lot of baggage.

Reply:
Yup, I think updating it falls more into a later phase though! I'd like to get new TUF just happily running alongside the current stuff before doing larger refactors.